client/packages/requisitions/src/RequestRequisition/DetailView/Footer/StatusChangeButton.tsx

```diff
@@ -141,13 +141,17 @@ export const StatusChangeButton = () => {
   const { selectedOption, getConfirmation, lines } = useStatusChangeButton();
   const isDisabled = useRequest.utils.isDisabled();
   const { userHasPermission } = useAuthContext();
-  const t = useTranslation('app');
-  const noLines = lines?.totalCount === 0;
+  const t = useTranslation(['app', 'replenishment']);
+  const cantSend =
+    lines?.totalCount === 0 ||
+    lines?.nodes.some(line => line.requestedQuantity === 0);
```

Comment:
```suggestion
    lines?.nodes.every(line => line.requestedQuantity === 0);
```

I think it should be fine to confirm with some placeholder lines? Just not only placeholder lines right?

Reply:
You're right XD